### ISSUE TITLE
GUI-less issue when running simulation with user-defined vector fields or scalar fields

### ISSUE THREAD

**Issue by maciekswat** [CompuCell3D/CompuCell3D#718]
here is the error
```commandline
#################################################
# CompuCell3D Version: 4.5.0 Revision: 6 
 Commit Label: 8190091
#################################################
GOT CUSTOM SETTINGS :  /Users/m/src/Demos-local/ExtraFields/Simulation/_settings.sqlite
<cc3d.core.CC3DSimulationDataHandler.CC3DSimulationData object at 0x127a82290>
Random number generator: MersenneTwister
Traceback (most recent call last):
  File "/Users/m/src/conda-build-repos/CompuCell3D/cc3d/CompuCellSetup/sim_runner.py", line 79, in run_cc3d_project
    exec(code, globals(), locals())
  File "/Users/m/src/Demos-local/ExtraFields/Simulation/ExtraFields.py", line 14, in <module>
    CompuCellSetup.run()
  File "/Users/m/src/conda-build-repos/CompuCell3D/cc3d/CompuCellSetup/simulation_setup.py", line 152, in run
    field_registry.create_fields()
  File "/Users/m/src/conda-build-repos/CompuCell3D/cc3d/core/FieldRegistry.py", line 177, in create_fields
    field_creating_fcn(field_name)
  File "/Users/m/src/conda-build-repos/CompuCell3D/cc3d/core/FieldRegistry.py", line 131, in create_vector_field
    ndarray_adapter = self.get_field_storage().createVectorFieldPy(self.dim, field_name)
  File "/Users/m/src/conda-build-repos/CompuCell3D/cc3d/core/FieldRegistry.py", line 258, in get_field_storage
    return pg.persistent_holder['field_storage']
KeyError: 'field_storage'
```

The simulation I ran is attached
[ExtraFields.zip](https://github.com/CompuCell3D/CompuCell3D/files/15284295/ExtraFields.zip)

@tjsego Question: have we ever supported GUI-less runs of simulations that have vector or scalar fields defined by the user?  Do you know a quick fix for it?

the command I used was the following:

```commandline

python -m cc3d.run_script --input /Users/m/src/Demos-local/ExtraFields/ExtraFields.cc3d -f 100

``` 

you would need to adjust paths accordingly

**Comment by tjsego** [comment]
Should be resolved by #710.

**Comment by maciekswat** [comment]
This should be fixed now
